### DISCUSSION TITLE
"cmake.ctestArgs" settings seems to be ignored

### DISCUSSION THREAD

**Discussion by RealLitb** [microsoft/vscode-cmake-tools#3385] [General]
I defined the following in my settings.json

    "cmake.ctestArgs": [
       "--output-on-failure"
    ],

In the default, it included  "-T test" in this list. However, The CMake tools extension still executes the tests as follows:

    [proc] Executing command: /usr/bin/ctest -T test ...

Why does it not apply the setting?

**Comment by gcampbell-msft** [comment]
I've tested on the most recent version of the extension and don't see this problem reproducing. I'll close this discussion based on that. However, if you still are hitting issues please create a GitHub issue for us to track better. Thanks!

**Reply by RealLitb** [reply]
Thanks for letting me know! I can confirm it works with the latest release.
